modules/ROOT/pages/index.adoc

```diff
@@ -118,13 +138,59 @@ image::create-database-access.png[Creating database access]
 
 . To set up a secure connection, you will need to retrieve the root certificate for your Capella instance.
 +
-[lowerroman]
 .. Click on btn:[Security Certificate] in the left menu.
 .. Click on btn:[Download] to transfer a copy of the root certificate to your local machine.
 +
 image::download-root-certificate.png[]
 --
 
+Capella Columnar::
++
+--
+
+For details on creating a Capella Columnar instance,
+first see xref:columnar:admin:prepare-project.adoc[Creating a Capella Columnar Cluster].
+
+[#capella-columnar-connection-string]#You will need to access the Capella administration console to get the connection string for your columnar database.#
+
+. Sign in to your Capella instance as an Administrator.
+. Select menu:Columnar[] from the top-level page menu.
+. Select your Columnar cluster from the list.
+. Select menu:Settings[] from the top-level page menu.
++
+image::select-columnar-cluster-settings.png[]
++
+. Select menu:Connection String[] from the left-hand menu.
++
+image::get-capella-columnar-connection-string.png[]
++
+. Make a note of the connection string.
+
+Next, you need to add the IP address of the machine from which you are running Power BI,
+ so that Capella will allow the machine to access the columnar data.
+
+. Click on menu:Allowed IP Addresses[] in the left-hand menu.
+. Click the btn:[Add Allowed IP] button.
+
++
+image::capella-columnar-add-allowed-ip.png[]
++
+. Enter the IP address of the Power BI host machine.
++
+TIP: You can use the btn:[Add Current IP Address] button to fill in the address of the machine currently running the web console.
+
+[#capella-columnar-user-account]#Now, you will need to create a user account for Power BI to access the columnar data.#
+
+. Return to the menu:Settings[] page, then click on menu:Access Control[] in the left-hand menu.
+. Click on btn:[Create Account].
+. Add a `username` and `password` for the new account.
+. Make sure you set assign  `sys_data_reader` to the list of roles.
```

Comment:
This depends. The default role that gives access to all the TAVs globally is sys_view_reader (https://couchbasecloud.atlassian.net/browse/AV-81710).
 Apart from this, the access accounts can be GRANTed privileges and also be able to query views from Power BI.

Reply:
Fair enough, but I'd rather just keep it simple, so the user doesn't have to go off on a tangent with security privileges.